katacoda/scenarios/app-developer-guide/specify-buildpacks.md

```diff
@@ -1,4 +1,4 @@
-
+# Specify buildpacks
```

Comment:
I didn't understand the reasoning for this but it seems like a few files have double H1 headings. 🤷🏽